config/application.rb

```diff
@@ -22,5 +22,8 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    # Which is the lowest balance you should be ashamed of.
+    config.shameful_balance = 50 # In eurocents!
```

Comment:
Thanks for linking the correct way of configuring things, I was looking for this, still.